config/kernelci.conf

```diff
@@ -20,7 +20,9 @@ output: /home/kernelci/output
 
 [notifier]
 
-[kcidb]
+[send_kcidb]
+kcidb_topic_name: playground_kcidb_new
+kcidb_project_id: kernelci-production
```

Comment:
Not sure this is the right project id, first of all the pipeline is actually running on staging.  Maybe `kernelci-api` would be better at this stage to differentiate it from the data sent by the current actual production / staging setup which uses `kernelci_backend`.

Reply:
Updated the project ID.